src/Network/Wai/SAML2.hs

```diff
@@ -104,7 +105,7 @@ saml2Callback cfg callback app req sendResponse = do
     let path = rawPathInfo req
 
     -- check if we need to handle this request
-    if | path == saml2AssertionPath cfg && isPOST req -> do
+    if path == saml2AssertionPath cfg && isPOST req then do
```

Comment:
I suppose this change from a multi-way if means that extension isn't required here. I am not sure why I went for one originally (perhaps I intended to have more cases checked).